src/wasm/wasm-binary.cpp

```diff
@@ -4689,7 +4709,7 @@ void WasmBinaryBuilder::visitDrop(Drop* curr) {
 
 void WasmBinaryBuilder::visitRefNull(RefNull* curr) {
   BYN_TRACE("zz node: RefNull\n");
-  curr->finalize();
+  curr->finalize(getHeapType());
```

Comment:
@aheejin, this changes the parsing of `ref.null` instructions to expect a type parameter. Do we ever emit `ref.null` in LLVM? If so, this probably needs to be coordinated with an LLVM change.

Reply:
Looks like we don't support `ref.null` instructions at all in LLVM, much less emit them.